tests/endpoint/ping_endpoint.test.js

```diff
@@ -0,0 +1,52 @@
+const createServer = require('../../server.js').createServer;
+const got = require('got');
+const assert = require('assert');
+
+const port = process.env.PORT_TEST || 9373;
+let server;
+let testResponse;
+
+
+before( async () => {
+    server = createServer(port);
+    testResponse = await got("http://localhost:"+port+"/v1/ping"); 
```

Comment:
Personlig synes jeg det er lettere å lese slike strenger dersom man bruker [String Interpolation](https://dmitripavlutin.com/string-interpolation-in-javascript/). Da kan man skrive denne linjen som,
```javascript
testResponse = await got(`http://localhost:${port}/v1/ping`); 
```

Hva tenker du selv?

Merk bruk av ` og ikke ' symbolet når man skal benytte String Interpolation.

Reply:
Enig med deg, egentlig gammel vane, og det er ikke så galt når du kun skal ha inn en variabel der.. men blir fort uoversiktelig med flere, så dette kan jeg endre